### DISCUSSION TITLE
Some missing potential upgrade mods

### DISCUSSION THREAD

**Discussion by ReinhardtGore** [xanthics/PoE_Weighted_Search#29] [Ideas]
Hey, first off love this tool, have been using it for min maxing my jewels and wands. 
I noticed though that a missing mod from the pool for wands is "Hatred has #% increased Aura Effect". Is there a reason this mod is missing? I was comparing some end game wands for the build and noticed that this mod is not included in the generator.

**Comment by xanthics** [comment]
Every mod that is considered has to have it's value sent from pob via pob-item-tester, which leads to longer and longer query strings.  I'm in the process of redesigning how mods are passed to the site but that work is being delayed by irl things and complications such as versioning(of query strings) with backwards compatibility.  It's being worked on but there is not an eta (yet).

**Reply by xanthics** [reply]
Update on this.  I have the initial support for array weights implemented on a test page, waiting on Item Tester now.

**Comment by norecha** [comment]
What about "Nearby Enemies have #% to Fire Resistance"? I feel like that should show up with "damage penetrates X fire resistance"

**Reply by xanthics** [reply]
It may be helpful to read [Resist Pen](https://pathofexile.gamepedia.com/Resistance_penetration) as reducing resistance and penetrating resistance are not the same thing.  One of the important differences being that pen only applies to hits.

As an effect of this it also means more mods being passed via query string so implementing it will also be gated by the finalized implementation of using an array for weights.  Which is to say I'll have to wait to revisit this.

**Comment by xanthics** [comment]
Site now has support for several more mods as well as easy to access, although perhaps difficult to browse, lists of mods that are supported and on what bases.  You will need to wait for an update to PoB-Item-Tester before the tool will be able to autofill the new mods.
